### PR TITLE
Adds bower to the site

### LINKED ISSUE #209: Add bower to site

### PR DESCRIPTION
**Pull request by pitchinvasion**
I've had to move some stuff around as we want bower to have the css compiled as well
as in sass form, so we're using the Bourbon/Neat gem now

I need to register the package still, going to wait until everyone's happy with us open sourcing

Closes #209

### CODE REVIEW

**Comment by pitchinvasion** [comment]
So quite a bit of moving around to get this working. In short, we no longer have any bower dependencies, it's easier just to put things into a vendor folder inside of stylesheets/js. We are also now using the neat/bourbon gem instead.

Sass files are now in the sass directory and when you run `rake publish:bower` it will ask you to enter a version number and will then compile the sass directory, check in all files and create a new git tag/release which will update bower. It sounds complex but is actually quite simple, I've documented it in the readme.

Let me know what you think, the good news is I've already now got all the bower files coming into another repo hopefully it won't be too painful to keep things synced

**Comment by roidriscoll** [comment]
:shipit:

File: Rakefile [hunk 1]
```diff
@@ -1,7 +1,33 @@
 require 'middleman-gh-pages'
+require 'json'
 
 namespace :assets do
   task :precompile do
     sh 'middleman build --verbose'
   end
 end
+
+desc "Publishes to github pages and publishs to bower version and publishs."
```
Comment (spike01) [inline Rakefile]: ???

Comment (makers-dev) [inline Rakefile]: Oops that's embarassing, clearly wasn't pairing/thinking at that point

**Comment by pitchinvasion** [comment]
So can I merge this spike?

**Comment by spike01** [comment]
MUURGE

**Pull request merged**